db/migrate/20180717215603_create_active_admin_comments.rb

```diff
@@ -0,0 +1,17 @@
+class CreateActiveAdminComments < ActiveRecord::Migration::Current
+  def self.up
+    create_table :active_admin_comments do |t|
```

Comment:
In activeadmin, we have comments for the administrators.
In this time, we are not going to need them, so we can remove them like this:
https://activeadmin.info/1-general-configuration.html#comments

After changing the configuration of the activeadmin, we should remove this migration doing like this:
First doing rollback like here: https://medium.com/into-the-forest/rails-migrations-tricks-guide-code-cheatsheet-included-dca935354f22 in the point 3, and then we can remove the file :)